src/pages/guides/index.md

```diff
@@ -251,7 +251,8 @@ This feature is scheduled to be rolled out on 10th February, 2022
 For a more robust and reliable verification, Adobe I/O Events adds below security validations for events delivered to your webhook. 
 
 - sending additional field of recipient_client_id as part of your event payload
```

Comment:
Should we use complete sentences as these bullets are not seamlessly following with the previous line. Something like -
* Adobe I/O Events sends an additional field of `recipient_client_id` as part of your event payload
* The event payload is signed digitally using a fixed public/private key pair generated by Adobe I/O Events. The digital signature is sent as request header to your webhook.
* I/O Events sends the relative paths of public keys, which are served from our fixed adobe domain adobeioevents.com.

Reply:
+1

Reply:
done